### PR TITLE
Added ProviderOperations endpoint and fixed description warnings

### PR DESCRIPTION
**Pull request by ajay-arora**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [ ] My spec meets the review criteria:
  - [ ] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [ ] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/devtestlabs/resource-manager/Microsoft.DevTestLab/stable/2016-05-15/DTL.json [hunk 1]
```diff
@@ -58,7 +74,7 @@
             "schema": { "$ref": "#/definitions/CloudError" }
           }
         },
-        "security": [ { "oauth2": [ "user_impersonation" ] } ],
+        "security": [ { "azure_auth": [ "user_impersonation" ] } ],
```
Comment (marstr) [inline specification/devtestlabs/resource-manager/Microsoft.DevTestLab/stable/2016-05-15/DTL.json]: [Taking a look at the Redis swagger](https://github.com/Azure/azure-rest-api-specs/blob/b9062411a7ac5b5dc3a8840b5c5e0f8200e9a8d3/specification/redis/resource-manager/Microsoft.Cache/stable/2017-10-01/redis.json#L25), I believe that in addition to renaming the security line from `oauth2` to `azure_auth`, one must define a "securityDefintions" clause.

More info here from the spec: https://swagger.io/specification/#securitySchemeObject

File: specification/devtestlabs/resource-manager/Microsoft.DevTestLab/stable/2016-05-15/DTL.json [hunk 2]
```diff
@@ -10,6 +10,22 @@
   "consumes": [ "application/json" ],
   "produces": [ "application/json" ],
   "paths": {
+    "/providers/Microsoft.DevTestLab/operations": {
+      "get": {
+        "tags": [ "ProviderOperations" ],
```
Comment (marstr) [inline specification/devtestlabs/resource-manager/Microsoft.DevTestLab/stable/2016-05-15/DTL.json]: In all of the other specs I'm looking at, the tag is just `Operations` not `ProviderOperations`.

**Review by marstr — CHANGES_REQUESTED** [review]

File: specification/devtestlabs/resource-manager/Microsoft.DevTestLab/stable/2016-05-15/DTL.json (hunk 1 above)
Comment (ajay-arora) [reply specification/devtestlabs/resource-manager/Microsoft.DevTestLab/stable/2016-05-15/DTL.json]: yeah, i already have that in line 8717

File: specification/devtestlabs/resource-manager/Microsoft.DevTestLab/stable/2016-05-15/DTL.json (hunk 2 above)
Comment (ajay-arora) [reply specification/devtestlabs/resource-manager/Microsoft.DevTestLab/stable/2016-05-15/DTL.json]: `Operations` are already used in the end point for polling long running operations. I think `ProviderOperations` makes it clear.

Comment (marstr) [reply specification/devtestlabs/resource-manager/Microsoft.DevTestLab/stable/2016-05-15/DTL.json]: @ravbhatnagar, do you have thoughts on the name of the Operations group? It sounds like we're sorta flexible here, which is fine with me. I just want to make sure that's true though.

File: specification/devtestlabs/resource-manager/Microsoft.DevTestLab/stable/2016-05-15/DTL.json (hunk 1 above)
Comment (marstr) [reply specification/devtestlabs/resource-manager/Microsoft.DevTestLab/stable/2016-05-15/DTL.json]: Excellent, thanks for the link. Sorry I didn't see it!

**Review by marstr — COMMENTED** [review]
Looks good from my perspective, I'll approve one @ravbhatnagar or @simongdavies have approved.

File: specification/devtestlabs/resource-manager/Microsoft.DevTestLab/stable/2016-05-15/DTL.json (hunk 2 above)
Comment (ravbhatnagar) [reply specification/devtestlabs/resource-manager/Microsoft.DevTestLab/stable/2016-05-15/DTL.json]: @marstr - Do you mean the operationId? or the tags value. The operationId should be consistent between teams and I am not sure how tags is used. So I dont have a recommendation on that.

**Comment by ravbhatnagar** [comment]
Looks good.

File: specification/devtestlabs/resource-manager/Microsoft.DevTestLab/stable/2016-05-15/DTL.json (hunk 2 above)
Comment (ajay-arora) [reply specification/devtestlabs/resource-manager/Microsoft.DevTestLab/stable/2016-05-15/DTL.json]: I think its used by the autorest tool to segregate the generated code

**Review by marstr — APPROVED** [review]

**Pull request merged**